packages/dashboard/src/features/database/pages/__tests__/SQLEditorPage.test.tsx

```
@@ -0,0 +1,344 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SQLEditorPage from '../SQLEditorPage';
```

ESLint

[error] '../SQLEditorPage' import is restricted from being used by a pattern. Use relative imports only within the same folder. Use dashboard #... aliases for imports outside the current folder.

(no-restricted-imports)